contracts/SortitionPool.sol

```diff
@@ -30,6 +30,10 @@ contract SortitionPool is SortitionTree, Rewards, Ownable, IReceiveApproval {
 
   bool public isLocked;
 
+  event IneligibleForRewards(uint32[] ids, uint256 until);
+
+  event EligibilityRestored(address indexed operator, uint32 indexed id);
```

Comment:
Actually... maybe `RewardEligibilityRestored` to not confuse with v1 situation when someone was not *eligible* to join the sortition pool. WDYT?

Reply:
Agreed and addressed.